packages/wallet-service/tests/txProcessor.test.ts

```diff
@@ -1,441 +1,19 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
-import { Logger } from 'winston';
-import firebaseMock from '@tests/utils/firebase-admin.mock';
-import { mockedAddAlert } from '@tests/utils/alerting.utils.mock';
-import hathorLib from '@hathor/wallet-lib';
-import eventTemplate from '@events/eventTemplate.json';
-import tokenCreationTx from '@events/tokenCreationTx.json';
-import {
-  getLatestHeight,
-  getTokenInformation,
-  fetchTx,
-  getTxOutput,
-  getWalletTxHistory,
-} from '@src/db';
-import * as Db from '@src/db';
 import * as txProcessor from '@src/txProcessor';
-import { closeDbConnection, getDbConnection, isAuthority } from '@src/utils';
 import { NftUtils } from '@wallet-service/common/src/utils/nft.utils';
-import {
-  XPUBKEY,
-  AUTH_XPUBKEY,
-  addToAddressTable,
-  addToAddressBalanceTable,
-  addToUtxoTable,
-  addToWalletTable,
-  addToWalletBalanceTable,
-  cleanDatabase,
-  checkUtxoTable,
-  checkAddressTable,
-  checkAddressBalanceTable,
-  checkAddressTxHistoryTable,
-  checkWalletBalanceTable,
-  createOutput,
-  createInput,
-  addToAddressTxHistoryTable,
-  addToWalletTxHistoryTable,
-} from '@tests/utils';
 import { getHandlerContext, nftCreationTx } from '@events/nftCreationTx';
-import * as pushNotificationUtils from '@src/utils/pushnotification.utils';
-import * as commons from '@src/commons';
-import { Context } from 'aws-lambda';
-import { StringMap, WalletBalanceValue } from '@src/types';
-import { Severity } from '@wallet-service/common/src/types';
-import createDefaultLogger from '@src/logger';
+
+const defaultLogger = {
+  error: jest.fn(),
+  warn: jest.fn(),
+  info: jest.fn(),
+}
 
 jest.mock('@src/logger', () => ({
   __esModule: true,
-  default: () => ({
-    error: jest.fn(),
-    warn: jest.fn(),
-    info: jest.fn(),
-  })
+  default: () => defaultLogger,
 }));
 
-const logger = createDefaultLogger();
-
-logger.warn = jest.fn();
-logger.error = jest.fn();
-logger.info = jest.fn();
-
-const mysql = getDbConnection();
-const blockReward = 6400;
-const OLD_ENV = process.env;
-
-beforeEach(async () => {
-  await cleanDatabase(mysql);
-});
-
-beforeAll(async () => {
-  // modify env so block reward is unlocked after 1 new block (overrides .env file)
-  jest.resetModules();
-  process.env = { ...OLD_ENV };
-  process.env.BLOCK_REWARD_LOCK = '1';
-  firebaseMock.resetAllMocks();
-});
-
-afterAll(async () => {
-  await closeDbConnection(mysql);
-  // restore old env
-  process.env = OLD_ENV;
-});
-
-/*
- * In an unlikely scenario, we can receive a tx spending a UTXO that is still marked as locked.
- */
-test.skip('spend "locked" utxo', async () => {
```

Comment:
We're handling transactions differently, we receive them in order, so this will never happen